CDF Election Process.md

```diff
@@ -0,0 +1,133 @@
+# CDF Election Process
+
+
+## Election Process
+
+Duration: 1 month
+
+
+### Announcing an Election Cycle
+
+Election cycle consists of 
+
+
+
+* Appointment of Election Officer
```

Comment:
Does it really happen in practice? IIUC @micmarti85 you de-facto hold this role for all the elections. It's fine with me, but maybe we need to be explicit who makes the appointment

Reply:
I actually served as a community member in the past so [it happened in practice](https://github.com/cdfoundation/toc/tree/master/elections/2021#election-officers) but the role and its place in the process were not documented.

Regarding who makes the appointment - we documented the role and who makes the appointment (TOC) [here](https://github.com/cdfoundation/foundation/pull/399/files#diff-76a57d1404b8b0c478f068f490ce6168d9a0893e75b75420f5fce5c1e1552c88R199) but it would be good to add it to this doc as well like *Appointment of Election Officer by TOC*.

Reply:
@oleg-nenashev does this answer your question?